### PR TITLE
Add Saleor to available projects

### PR DESCRIPTION
**Pull request by Pacu2**
This PR adds Saleor (https://getsaleor.com/) to available projects. 
Saleor is high performance, graphql-first e-commerce Storefront, created with Python, Django, Apollo and ReactJS

### CODE REVIEW

File: _data/projects/saleor.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Saleor
+desc: Saleor is a high-performance, GraphQL-first e-commerce solution created with Python and Django. https://getsaleor.com/
+site: https://github.com/mirumee/saleor/
+tags:
+- Python
+- Django
+- GraphQL
+- React
+- e-commerce
+- GraphQL
```
Comment (ritwik12) [inline _data/projects/saleor.yml]: @Pacu2 This tag is being used twice due to which travis is failing.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/saleor.yml (hunk 1 above)
Comment (Pacu2) [reply _data/projects/saleor.yml]: Right, good catch, thanks

**Review by ritwik12 — APPROVED** [review]

File: _data/projects/saleor.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- React
+- e-commerce
+- web
+- storefront
+- Typescript
+- Apollo
+- Docker
+upforgrabs:
+  name: help wanted
+  link: https://github.com/mirumee/saleor/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
Comment (ritwik12) [inline _data/projects/saleor.yml]: @Pacu2 Please change this to https://github.com/mirumee/saleor/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/saleor.yml (hunk 2 above)
Comment (Pacu2) [reply _data/projects/saleor.yml]: Done

Comment (ritwik12) [reply _data/projects/saleor.yml]: Great

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@Pacu2 Thanks!!

**Pull request merged**